model-optimizer/extensions/front/mxnet/gluoncv_ssd_anchors.py

```diff
@@ -90,22 +68,28 @@ class SsdAnchorsReplacer(FrontReplacementPattern):
     """
 
     enabled = True
-    graph_condition = [lambda graph: graph.graph['fw'] == 'mxnet' and graph.graph['cmd_params'].enable_ssd_gluoncv]
+    graph_condition = [lambda graph: graph.graph['cmd_params'].enable_ssd_gluoncv]
 
     def run_after(self):
-        return [SsdAnchorReshape, SsdPatternDetectionOutputReplacer]
+        return [SsdPatternDetectionOutputReplacer]
 
     def pattern(self):
         return dict(
             nodes=[
                 ('slice_like', dict(op='slice_like')),
                 ('reshape0', dict(op='Reshape')),
+                ('reshape1', dict(op='Reshape')),
+                ('reshape2', dict(op='Reshape')),
+                ('reshape3', dict(op='Reshape')),
                 ('concat', dict(op='Concat')),
                 ('detection_output', dict(op='DetectionOutput'))
             ],
             edges=[
                 ('slice_like', 'reshape0'),
-                ('reshape0', 'concat', {'in': 0}),
+                ('reshape0', 'reshape1'),
```

Comment:
You will get 6 matches for one Concat with varying Reshape transposition. Please use ports to fix that.

Reply:
@jane-intel But I am using it in string 92

Reply:
Sorry, missunderstood